Frigate Camera Notifications/Beta

```diff
@@ -809,212 +881,336 @@ variables:
+        - alias: "Notifications enabled for object"
+          condition: template
+          value_template: "{{ not labels|length or labels|select('in', objects)|list|length > 0 }}"
+        - "{{ not this.attributes.last_triggered or (now() - this.attributes.last_triggered).seconds > cooldown }}"
+        - "{{ not disable_times|length or not now().hour in disable_times|map('int')|list }}"
+        - "{{ not zone_only or (not zone_multi and zones|select('in', after_zones)|list|length > 0) or (zone_multi and after_zones|length > 0 and zones |reject('in', after_zones) |list |length == 0) }}"
+        - "{{ not initial_home }}"
+        - "{{ not state_only or states(input_entity) in states_filter }}"
+        - "{{ custom_filter }}"
+        - "{{ type == 'new' or type  == 'end' or update_thumbnail }}"
+actions:
+  - if:
+      - condition: !input master_condition
+    then:
+      - choose:
+          - alias: "Silence New Object Notifications"
+            conditions:
+              - condition: trigger
+                id: silence
+            sequence:
+              - action: automation.turn_off
+                target:
+                  entity_id: "{{ this.entity_id }}"
+                data:
+                  stop_actions: false
+              - delay:
+                  minutes: !input silence_timer
+              - action: automation.turn_on
+                target:
+                  entity_id: "{{ this.entity_id }}"
+          - alias: "Custom Action Manual"
+            conditions:
+              - condition: trigger
+                id: custom
+            sequence: !input "custom_action_manual"
+          - alias: "Frigate Event"
+            conditions:
+              - condition: trigger
+                id: "frigate-event"
+            sequence:
+              - variables:
+                  event: "{{ trigger.payload_json }}"
+                  detections: "{{ event['after']['data']['detections'] }}"
+                  review_id: "{{event['after']['id']}}"
+                  id: "{{ detections[0] }}"
+                  objects: "{{ event['after']['data']['objects'] }}"
+                  sub_labels: "{{ event['after']['data']['sub_labels'] }}"
+                  label: >-
+                    {% set data = namespace(labels=[]) %} 
+                    {% for obj in objects %}
+                      {% if "-verified" in obj %}
+                      {% else %}
+                        {% set data.labels = data.labels + [obj] %}
+                      {% endif %}
+                    {% endfor %} {% set data.labels = data.labels + sub_labels %} 
+                    {{ data.labels | unique | list | sort | join(", ") | title }}
+
+                  # Dynamic Variables per event
+                  initial_home: "{{ presence_entity != '' and is_state(presence_entity, 'home') }}"
+                  entered_zones: "{{ trigger.payload_json['after']['entered_zones'] | default([]) }}"
```

Comment:
I'm new to blueprints, but I think this should be:

`entered_zones: "{{ trigger.payload_json['after']**[''data'']**['entered_zones'] | default([]) }}"`

Reply:
The frigate mqtt messages don't have that data key in the middle so it's not needed here.

Reply:
Interesting. I'm running 0.15 beta1 and I do have/need data here. I wonder if there was a schema change. Debug wasn't showing any zones without adding this. Further up in the blueprint, data is present and those are working for me.

Reply:
I see the data key present in the v14 docs as well.

Reply:
Yep im an idiot. I referenced the events messages when i made this update and ruined all the new structure for reviews. Thank you.